src/src/ManagerSync.php

```diff
@@ -92,18 +92,18 @@ public function enforce_latest_version(): void {
 		}
 
 		$latest_version = $this->environment->get_cache()->get_manager_sync_data( 'latest_cli_version' );
-		$enforce_latest = $this->environment->get_cache()->get_manager_sync_data( 'enforce_latest' );
+		$minimum_cli_version = $this->environment->get_cache()->get_manager_sync_data( 'minimum_cli_version' );
 
 		if ( version_compare( $current_version, $latest_version, '<' ) ) {
-			if ( $enforce_latest ) {
+			$this->output->writeln( '<comment>There\'s a new version of the QIT CLI available!</comment>' );
```

Comment:
Can we update the wording here to have a red background so it is more prominent among the other text in the terminal?

Reply:
Thanks for the suggestion. What do you think about these?

**Using a version below the minimum set by the Manager:**
![image](https://github.com/woocommerce/qit-cli/assets/9341686/0619f8ad-b306-4ce9-b5de-9c7177dc6c38)

**Using an outdated, but still supported version:**
![image](https://github.com/woocommerce/qit-cli/assets/9341686/da34f930-f5eb-4a5b-bcc1-57469485f981)

Reply:
Thanks for making the change! Users definitely won't miss this